styles.css

```diff
@@ -0,0 +1,642 @@
+    color: white;
+    /* font-size: 1.5rem; */
+    font-weight: 600;
+    text-align: center;
+}
+#email-link:hover{
+    #copy-button{
+        display: block;
+        color: white;
+        z-index: 10;
+    }
+}
+#copy-info{
+    display: none;
+    color: white;
+    font-size: 0.8rem;
+    font-weight: 100;
+    margin: 0;
+    text-align: center;
+}
+
+#contact .title, #coming-soon .title{
+    padding-top: 50px;
+    margin: 2% 0 0;
+}
+
+
+/*Footer*/
+
+footer {
+    margin-top: -20px;
+    display: flex; /*Keeps all items on singular line*/
+    justify-content: end; /*Equally spaces out div elements in nave bar*/
+    border-radius: var(--curve) var(--curve) 0px 0px;
+    background-color: var(--tertiary);
+    color: white;
+}
+
+#reference {
+   font-size: 0.5rem;
+   padding: 0.5rem;
+   font-weight: 550;
+   color: var(--white);
+   text-align: center;
+}
+
+
+/*Media adjustments*/
+
+@media screen and (max-width: 850px) {
+  #home h2{
+    font-size: 1rem;
+    }
+  #contact .home-content{
+    /* margin: 0 0% auto 0%; */
+    width: 90%;
+  }
+  section{
+    padding-bottom: 3%;
+  }
+  nav ul a, nav ul li, #name, #nav-contact, nav div a {
+    font-size: 1.2rem;
+    padding: 0.2rem;
+    }
+}
+
+@media screen and (min-width: 1500px) {
+  #home h2{
+    font-size: 2rem;
+    }
+  .home-content{
+    width: 60%;
+  }
+  section{
+    padding: 7.5%;
+    justify-content: space-between;
+    gap: 15%;
+    }
+  .title {
+    margin-top: 0;
+    }
+  .summary {
+    justify-content: space-evenly;
+    }
+  .summary-box {
+    justify-content: space-between;
+    }
+  #projects-page .summary-box{
+    margin: 0 5% 5%;
+    justify-content: space-between;
+    }
+  #contact .title{
+    margin-top: 5%;
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  section{
+    height: auto;
+    /* min-block-size: auto; */
+  }
+  #name, #nav-contact {
+     width: auto;
+}
+  /* Style the navigation menu */
+  nav {
+      overflow: hidden;
+      flex-direction: column;
+      position: fixed;
+      border-radius: 0 0 0 var(--curve);
+    }
+    nav ul li {
+    float: none;
+    }
+    /* Hide the links inside the navigation menu (except for logo/home) */
+  nav #nav-list, nav #nav-contact {
+      display: none;
+      padding-left: 3%;
+      text-align: left;
+      padding-top: 0;
+      font-size: 1.5rem;
+    }
+  nav #name{
+      padding-left: 5%;
+      font-size: 2rem;
+    }
+    /* Style navigation menu links */
+  nav a {
+      display: block;
+    }
+    /* Style the hamburger menu */
+  nav #menu-icon {
+    background: white;
+    display: flex;
+    position: absolute;
+    width: 10%;
+    align-items: center;
+    justify-content: center;
+    right: 0;
+    padding: 0 1%;
+    height: 45px;
+    }
+  #burger {
+    font-size: xx-large;
+    color: var(--tertiary);
+    }
+  #home, #contact {
+    max-height: 100vh;
+    height: 100vh;
+  }
+  #projects{
+    max-height: 1400px;
```

Comment:
This could cause problems on smaller screens

Reply:
Removed as no longer needed